x-pack/plugins/security_solution/public/common/components/event_details/__mocks__/index.ts

```diff
@@ -9,6 +9,12 @@ export const mockAlertDetailsData = [
   { category: 'process', field: 'process.name', values: ['-'], originalValue: '-' },
   { category: 'process', field: 'process.pid', values: [0], originalValue: 0 },
   { category: 'process', field: 'process.executable', values: ['-'], originalValue: '-' },
+  {
+    category: 'process',
+    field: 'process.entry_leader.entity_id',
```

Comment:
Let's just confirm with the AWP team that this value is the best one for us to use. I think so, but want to make sure nothing has changed

Reply:
Confirmed with them offline 👍